docs/interfaces/json-api.md

```
@@ -186,7 +179,7 @@ stopY | 1 to matrix height | Stop row from top-left corner of matrix. _(availab
 len | 0 to info.leds.count | Length of the segment (_stop_ - _start_). _stop_ has preference, so if it is included, _len_ is ignored.
 grp | 0 to 255 | Grouping (how many consecutive LEDs of the same segment will be grouped to the same color)
 spc | 0 to 255 | Spacing (how many LEDs are turned off and skipped between each group)
-of | -len+1 to len | Offset (how many LEDs to rotate the virtual start of the segments, available since 0.13.0)
+of | -len+1 to len | Offset (how many LEDs to rotate the virtual start of the segments
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_only; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)